web/src/components/Cart/Cart.js

```diff
@@ -94,12 +95,24 @@ const ShoppingCartButton = styled(Button)`
     `}
 `
 
-const CartItem = ({ image, quantity, name }) => {
+const RemoveCartButton = styled(Button)`
```

Comment:
Could we call it `RemoveFromCartButton` just for consistency?

```suggestion
const RemoveFromCartButton = styled(Button)`
```

Reply:
Sounds good. I've updated the name.